### ISSUE TITLE
Detection quality

### ISSUE THREAD

**Issue by Ddk007** [EdjeElectronics/Train-and-Deploy-YOLO-Models#5]
I'm experimenting with the detection of micro-objects that are visible through a microscope. Multicolored microparticles in the field of view of the microscope are detected and accurately classified by a neural network. However, some objects that are clearly visible remain undetected. The photo below shows an example of a neural network that was trained on 399 images.

I have tried various settings, such as different epochs and image sizes, as well as different versions of YOLO, but about 10% of objects are still undetectable. What can be done to improve the accuracy of the detection?

![Image](https://github.com/user-attachments/assets/cc57da2b-7af6-4ff1-b20d-c8d310393da4)

**Comment by EdjeElectronics** [comment]
Cool project! But those objects are too small to be detected by YOLO. The YOLO models generally need an object to be at least 20x20 pixels for it to see them. That's AFTER the original image has already been shrunk to the resolution of the input model (640x640 for default YOLO models). 

If the camera supports a high resolution, look into using SAHI to slice each frame into multiple sections so you can perform inference on each section. It's a bit complicated, though.

Here's a good article on the topic: https://blog.roboflow.com/detect-small-objects/
